ci/publish.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+const AWS = require('aws-sdk');
+const { App } = require("@octokit/app");
+const Octokit = require("@octokit/rest");
+
+const BINARY_PATH = './target/release/hecate';
+
+async function getCredentials(secretPrefix) {
+    const sm = new AWS.SecretsManager({ region: 'us-east-1' });
+
+    const [
+        { SecretString: id },
+        { SecretString: installationId },
+        { SecretBinary: privateKey }
+    ] = await Promise.all([
+        sm.getSecretValue({ SecretId: `${secretPrefix}/app-id` }).promise(),
+        sm.getSecretValue({ SecretId: `${secretPrefix}/installation-id` }).promise(),
+        sm.getSecretValue({ SecretId: `${secretPrefix}/secret` }).promise()
+    ]);
+
+    return { id, installationId, privateKey };
+}
+
+async function buildOctokit(secretPrefix) {
+    const { id, installationId, privateKey } = await getCredentials(secretPrefix);
+
+    const app = new App({ id, privateKey });
+    return new Octokit({
+        async auth() {
+            const installationAccessToken = await app.getInstallationAccessToken({ installationId });
+            return `token ${installationAccessToken}`;
+        }
+    });
+}
+
+async function uploadBinary(octokit, uploadUrl) {
+    return octokit.repos.uploadReleaseAsset({
+        name: 'hecate',
+        file: fs.createReadStream(BINARY_PATH),
+        url: uploadUrl,
+        headers: {
+            'content-length': fs.statSync(BINARY_PATH).size,
+            'content-type': 'application/octet-stream'
+        }
+    });
+}
+
+async function createRelease(octokit) {
+    return octokit.repos.createRelease({
+        owner: 'mapbox',
+        repo: 'Hecate',
+        tag_name: process.env.CIRCLE_TAG,
+        prerelease: true
```

Comment:
This is small, but can we also automatically populate the title here and update the instructions in `CONTRIBUTING`? One less thing to fill in

Reply:
Good call - done!